.changeset/config.json

```
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "bigcommerce/catalyst" }
+  ],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "privatePackages": {
+    "version": true,
+    "tag": true
+  },
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
+}
```

What happens if we change it to something like this?
```suggestion
{
  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
  "changelog": "@changesets/cli/changelog",
  "commit": false,
  "fixed": [],
  "linked": [],
  "access": "public",
  "baseBranch": "main",
  "updateInternalDependencies": "patch",
  "ignore": ["@bigcommerce/docs", "@bigcommerce/components", "functional"]
}
```

---

The only reason I was hesitant on the `ignore` property is because of [this warning](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#ignore-array-of-packages):
> THIS FEATURE IS DESIGNED FOR TEMPORARY USE TO ALLOW CHANGES TO BE MERGED WITHOUT PUBLISHING THEM - If you want to stop a package from being published at all, set private: true in its package.json.

I think we want to keep the `privatePackages` prop for tag generation of core, right?

Also, the changelog being generated by `@changesets/changelog-github` is preferred because of the links to PR's 👍

---

>Also, the changelog being generated by @changesets/changelog-github is preferred because of the links to PR's 👍

Okay cool

>I think we want to keep the privatePackages prop for tag generation of core, right?

Most likely we'll remove `private` on core, unless I'm missing something.

---

See PR comment below. Marking this as resolved based on our last meeting 👍